### ISSUE TITLE
Incorrect file permissions for version 0.0.9

### ISSUE THREAD

**Issue by pfriel-bcs** [GoogleCloudPlatform/fluent-plugin-detect-exceptions#32]
# Problem description
When installing the fluent-plugin-detect-exceptions plugin v0.0.9 via "td-agent-gem install fluent-plugin-detect-exceptions:0.0.9" the plugin is installed with incorrect file permissions.  The files are all owned by root:root with 640 perms.  Standard for fluentd plugins is owned by root:root with 644 perms. The td-agent user is unable to read the plugin files so td-agent fails to start.

# OS info
```
root@hostnameredacted:~# cat /etc/lsb-release 
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=16.04
DISTRIB_CODENAME=xenial
DISTRIB_DESCRIPTION="Ubuntu 16.04.3 LTS"
```

# App info
## Perms after installing the plugin
```
root@hostnameredacted:/opt/td-agent/embedded/lib/ruby/gems/2.4.0/gems/fluent-plugin-detect-exceptions-0.0.9# find . -type f -exec ls -al {} \;
-rw-r----- 1 root root 39 Mar 19 18:08 ./Gemfile
-rw-r----- 1 root root 11323 Mar 19 18:08 ./LICENSE
-rw-r----- 1 root root 997 Mar 19 18:08 ./Rakefile
-rw-r----- 1 root root 1413 Mar 19 18:08 ./Gemfile.lock
-rw-r----- 1 root root 4449 Mar 19 18:08 ./lib/fluent/plugin/out_detect_exceptions.rb
-rw-r----- 1 root root 9469 Mar 19 18:08 ./lib/fluent/plugin/exception_detector.rb
-rw-r----- 1 root root 1244 Mar 19 18:08 ./fluent-plugin-detect-exceptions.gemspec
-rw-r----- 1 root root 5314 Mar 19 18:08 ./README.rdoc
-rw-r----- 1 root root 1395 Mar 19 18:08 ./CONTRIBUTING
-rw-r----- 1 root root 8749 Mar 19 18:08 ./test/plugin/test_out_detect_exceptions.rb
-rw-r----- 1 root root 22338 Mar 19 18:08 ./test/plugin/test_exception_detector.rb
-rw-r----- 1 root root 2617 Mar 19 18:08 ./test/plugin/bench_exception_detector.rb
-rw-r----- 1 root root 1359 Mar 19 18:08 ./test/helper.rb
```

## Installed package info
```
root@hostnameredacted:~# dpkg -l |grep td-agent
ii  td-agent                          3.1.1-0                                    amd64        Treasure Agent: A data collector for Treasure Data
```
```
root@hostnameredacted:~# td-agent-gem list |grep fluent
fluent-logger (0.7.1)
fluent-plugin-concat (2.1.0)
fluent-plugin-detect-exceptions (0.0.9)
fluent-plugin-dogstatsd (0.0.6)
fluent-plugin-elasticsearch (2.4.0)
fluent-plugin-flowcounter (1.3.0)
fluent-plugin-kafka (0.6.5)
fluent-plugin-multi-format-parser (1.0.0)
fluent-plugin-record-reformer (0.9.1)
fluent-plugin-rewrite-tag-filter (2.0.1)
fluent-plugin-route (1.0.0)
fluent-plugin-s3 (1.1.0)
fluent-plugin-td (1.0.0)
fluent-plugin-td-monitoring (0.2.3)
fluent-plugin-webhdfs (1.2.2)
fluentd (1.0.2)
```

**Comment by igorpeshansky** [comment]
Something like https://github.com/GoogleCloudPlatform/fluent-plugin-google-cloud/commit/27134d5b035ac81327811f050b1a22732165cceb should fix this. Please feel free to send a PR.

**Comment by mobilesfinks** [comment]
Open PR for fix this https://github.com/GoogleCloudPlatform/fluent-plugin-detect-exceptions/pull/35

**Comment by mobilesfinks** [comment]
@igorpeshansky please look at PR #35. Thank you.

**Comment by igorpeshansky** [comment]
Fixed by #35.
